ironmon_tracker/Drawing.lua

```diff
@@ -958,4 +958,14 @@ function Drawing.drawImageAsPixels(imageType, x, y, imageShadow)
 			end
 		end
 	end
+end
+
+--sets the form location relative to the game window
+--this function does what the built in forms.setlocation function supposed to do
+--currently that function is bugged and should be fixed in 2.9
+function Drawing.setFormLocation(handel,x,y)
+	local ribbonHight = 64 -- so we are below the ribbon menu 
+	local actualLocation = client.transformPoint(x,y)
+	forms.setproperty(Input.noteForm, "Left", client.xpos() + actualLocation['x'] )
+	forms.setproperty(Input.noteForm, "Top", client.ypos() + actualLocation['y'] + ribbonHight)
```

Comment:
Two things here:
1) This function accepts `handel` (typo?) as a parameter but doesn't use it anywhere. I suspect the `Input.noteForm` is supposed to be this parameter, which would allow using this function for other form controls that are rendered within the Tracker. There are a few more I'd like to extend this to.
2) The `Drawing.lua` file ideally handles anything that relates to drawing on the Bizhawk emulation screen, usually in the space reserved for the Tracker. As this function alters something else that is unrelated to drawing, can you move it to `Utils.lua`